components/Hero/Wave.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import './wave.module.css'
+
+export const Wave = () => {
+  return (
+    <div className='wave__content--desktop'>
+        <div className="header__wave--desktop">
+          <div className="header__wave" 
+            style={{height: '150px', overflow:'hidden'}}>
+              <svg viewBox="0 0 500 150" preserveAspectRatio="none" 
+              style={{height: '100%', width: '100%'}}>
+                <path d="M-4.05,-2.46 C288.77,292.59 244.16,-148.51 502.91,84.38 L506.67,-11.34 L-1.62,-16.27 Z" 
+                  style={{stroke: 'none', fill: '#132426'}}>
+                </path>
+              </svg>
+          </div>
+        </div>
+
+          <div className="header__wave--movil"> 
+           <div className="header__wave"
+            style={{height: '150px', overflow: 'hidden'}} >
+              <svg viewBox="0 0 500 150" preserveAspectRatio="none" 
+              style={{height: '100%', width: '100%'}}>
+                <path d="M-9.73,3.46 C185.75,75.49 322.83,81.41 511.02,1.49 L501.29,-16.27 L-1.62,-16.27 Z" 
+                  style={{stroke: 'none', fill: '#132426'}}>
+                </path>
+              </svg>
+          </div>
+          </div>  
+    </div>
```

Comment:
`    <div className='wave__content--desktop'>
        <div className="wave__hero">
          <div className="wave__hero">
              <svg viewBox="0 0 500 150" preserveAspectRatio="none">
                <path d="M-4.05,-2.46 C288.77,292.59 244.16,-148.51 502.91,84.38 L506.67,-11.34 L-1.62,-16.27 Z">
                </path>
              </svg>
          </div>
        </div>

          <div className="wave__hero--mobile"> 
           <div className="wave__hero">
              <svg viewBox="0 0 500 150" preserveAspectRatio="none">
                <path d="M-9.73,3.46 C185.75,75.49 322.83,81.41 511.02,1.49 L501.29,-16.27 L-1.62,-16.27 Z">
                </path>
              </svg>
          </div>
          </div>  
    </div>`

Reply:
can we please remove the inline styles and add it to the hero.css file ?

Reply:
Can we move the Wave component into it's own folder.

Wave/wave.js
Wave/wave.css

Reply:
can we please remove the inline styles and add it to wave.css file ?

- Which line?